doc/manual.rst

```diff
@@ -3678,6 +3678,47 @@ Is short for:
 
 
 
+Routines
+--------
+
+A routine is a symbol of kind: `proc`, `func`, `iterator`, `macro`, `template`, `converter`.
```

Comment:
While this is indeed the definition of a "routine" (except that you forgot to mention ``method``) the type bound operations cannot be any "routine", unsupported are: iterators, macros, methods, templates and converters.

Reply:
fixed; indeed, I verified that `template` didn't work